config/rootfs/chromiumos/scripts/build_board.sh

```diff
@@ -25,7 +28,14 @@ git config --global user.email "bot@kernelci.org"
 git config --global user.name "KernelCI Bot"
 git config --global color.ui false
 repo init --repo-url https://chromium.googlesource.com/external/repo --manifest-url https://chromium.googlesource.com/chromiumos/manifest --manifest-name default.xml --manifest-branch ${BRANCH}
```

Comment:
How about using parameters for the git URL, git branch and manifest file name here? Then we could use the kernelci-core URL with the chromeos.kernelci.org branch and the snapshot name including the release version (R100 etc.).

Reply:
Definitely, great idea, it looks way more neat like this.
Done, updated, will do testing by hand when branch get updated